myStyles.css

```diff
@@ -16,182 +16,1199 @@
+    margin-top: 8%;
+  }
+
+  .WorkTitle h2 {
+    white-space: nowrap;
+    font-size: 40px;
+    margin-left: 3%;
+    margin-right: 3%;
+  }
+
+  .WorkSeparation {
+    width: 100%;
+    align-self: center;
+    height: 1px;
+  }
+
+  .centerWork {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    padding-left: 10%;
+    padding-right: 10%;
+    height: 450px;
+  }
+
+  .worksCards_Container {
+    gap: 2.5%;
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(30%, 1fr));
+    margin-left: 5%;
+    margin-right: 5%;
+    margin-top: 5%;
+  }
+
+  .WorkImage {
+    display: flex;
+    justify-content: center;
+    width: 250%;
+  }
+
+  .WorkImage img {
+    width: 100%;
+    height: auto;
+    object-fit: contain;
+  }
+
+  .contextWork {
+    display: flex;
+    flex-direction: column;
+    margin-top: 7%;
+  }
+
+  .worksBody {
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+    border: 0;
+  }
+
+  .worksBody h3 {
+    font-size: 32px;
+    margin-left: 10%;
+    margin-bottom: 5%;
+  }
+
+  .worksBody p {
+    font-size: 16px;
+    margin-left: 10%;
+  }
+
+  .worksList ul {
+    margin-left: 10%;
+  }
+
+  .workButton {
+    display: flex;
+    width: auto;
+    align-self: flex-start;
+    margin-left: 10%;
+  }
+
+  .socialLinks_Bottom {
+    display: inline-block;
+    order: 1;
+    margin-right: 2%;
+    margin-left: -8%;
+  }
+
+  .horizontal-list {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .horizontal-list-bottom {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .card {
+    width: 240px;
+    height: 290px;
+    justify-content: end;
+    padding-bottom: 4%;
+    background-image: url("images/Work_Init/WorkCardBlk_Sml.png");
+  }
+
+  .card:hover {
+    background-image: url("images/Work_Init/WorkCard.png");
+  }
+
+  .card h2 {
+    font-size: 25px;
+    color: #fff;
+    white-space: nowrap;
+  }
+
+  .coment_container h2 {
+    text-align: left;
+    margin-top: 5%;
+  }
+
+  .comments h2 {
+    text-align: left;
+    font-size: 32px;
+    padding-left: 10%;
+    width: 60%;
+  }
+
+  .card:hover h2 {
+    display: none;
+  }
+
+  .card h3 {
+    color: #fff;
+    font-size: 32px;
+  }
+
+  .card:hover h3 {
+    display: none;
+  }
+
+  .card h4 {
+    color: #fff;
+    font-size: 32px;
+  }
+
+  .card:hover h4 {
+    display: none;
+  }
+
+  .card p {
+    font-size: 12px;
+    color: #fff;
+  }
+
+  .aboutMe p {
+    font-size: 20px;
+    max-width: 650px;
+    padding-bottom: 12%;
+    padding-left: 10%;
+    text-align: left;
+  }
+
+  .card:hover p {
+    display: none;
+  }
+
+  .card:hover ul {
+    display: none;
+  }
+
+  .card:hover button {
+    display: block;
+  }
+
+  .flexCardTitle {
+    display: none;
+  }
+
+  .worksListCard li {
+    font-size: 12px;
+  }
+
+  .workButtonCard {
+    display: none;
+  }
+
+  .aboutMe {
+    margin-top: 10%;
+    background-image: url("images/AboutMe/Background_Big.png");
```

Comment:
- [ ] You did a great job making the page responsive. :+1: I think this background image is positioned a bit lower when compared to the Figma design. Kindly push it to the top so it can be positioned to the right of the content. 🤝️
- [ ] Please also decrease the padding of the button to match the design. 😎️

**Your design**
![Screenshot from 2023-06-09 07-09-04](https://github.com/alexansaa/Personal_Portafolio/assets/17243325/87b4a094-32e5-4d26-bc50-e1c7b459388e)

**Figma design**
![Screenshot from 2023-06-09 07-09-56](https://github.com/alexansaa/Personal_Portafolio/assets/17243325/6da82e49-7815-476f-b60b-d04550584f3a)